templates/channels/channel_configuration.haml

```diff
@@ -510,6 +510,47 @@
       %code= ip_address
 
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To use your Firebase Cloud Messaging channel you'll have to POST to the following URLs with the parameters below."
+
+    .clearfix
+      .config-value
+        .name
+          -trans "Name:"
+        .value {{ object.name }}
+      .config-value
+        .name
+          -trans "Address/Key:"
+        .value {{ object.address }}
+      .config-value
+        .name
+          -trans "Notification title:"
+        .value {{ object.config_json.FCM_TITLE }}
+
```

Comment:
Well what are these used for? If there isn't any configuration the user has to do beyond hooking up the channel then we shouldn't be showing them things. They need to set a callback on the Firebase side though ya?